TAS.Avalonia/ViewModels/MainWindowViewModel.cs

```diff
@@ -67,27 +86,39 @@ public class MainWindowViewModel : ViewModelBase {
-        SimplifiedHitboxesCommand = ReactiveCommand.Create(SimplifiedHitboxes);
-        ShowActualCollideHitboxesCommand = ReactiveCommand.Create(ShowActualCollideHitboxes);
-
-        SimplifiedGraphicsCommand = ReactiveCommand.Create(SimplifiedGraphics);
-        ShowGameplayCommand = ReactiveCommand.Create(ShowGameplay);
-
-        CenterCameraCommand = ReactiveCommand.Create(CenterCamera);
-
-        InfoHudCommand = ReactiveCommand.Create(InfoHud);
-        InfoTasInputCommand = ReactiveCommand.Create(InfoTasInput);
-        InfoGameCommand = ReactiveCommand.Create(InfoGame);
-        InfoWatchEntityCommand = ReactiveCommand.Create(InfoWatchEntity);
-        InfoCustomCommand = ReactiveCommand.Create(InfoCustom);
-        InfoSubpixelIndicatorCommand = ReactiveCommand.Create(InfoSubpixelIndicator);
+        ToggleTriggerHitboxesCommand = ReactiveCommand.Create(ToggleTriggerHitboxes);
+        ToggleUnloadedRoomsHitboxesCommand = ReactiveCommand.Create(ToggleUnloadedRoomsHitboxes);
+        ToggleCameraHitboxesCommand = ReactiveCommand.Create(ToggleCameraHitboxes);
+        ToggleSimplifiedHitboxesCommand = ReactiveCommand.Create(ToggleSimplifiedHitboxes);
+        ToggleActualCollideHitboxesCommand = ReactiveCommand.Create(ToggleActualCollideHitboxes);
+
+        ToggleSimplifiedGraphicsCommand = ReactiveCommand.Create(ToggleSimplifiedGraphics);
+        ToggleGameplayCommand = ReactiveCommand.Create(ToggleGameplay);
+
+        ToggleCenterCameraCommand = ReactiveCommand.Create(ToggleCenterCamera);
+
+        ToggleInfoHudCommand = ReactiveCommand.Create(ToggleInfoHud);
+        ToggleInfoTasInputCommand = ReactiveCommand.Create(ToggleInfoTasInput);
+        ToggleInfoGameCommand = ReactiveCommand.Create(ToggleInfoGame);
+        ToggleInfoWatchEntityCommand = ReactiveCommand.Create(ToggleInfoWatchEntity);
+        ToggleInfoCustomCommand = ReactiveCommand.Create(ToggleInfoCustom);
+        ToggleInfoSubpixelIndicatorCommand = ReactiveCommand.Create(ToggleInfoSubpixelIndicator);
+        ToggleUnitOfSpeedCommand = ReactiveCommand.Create(ToggleUnitOfSpeed);
+
+        SetPositionDecimalsCommand = ReactiveCommand.CreateFromTask(SetPositionDecimals);
+        SetSpeedDecimalsCommand = ReactiveCommand.CreateFromTask(SetSpeedDecimals);
+        SetVelocityDecimalsCommand = ReactiveCommand.CreateFromTask(SetVelocityDecimals);
+        SetCustomInfoDecimalsCommand = ReactiveCommand.CreateFromTask(SetCustomInfoDecimals);
+        SetSubpixelIndicatorDecimalsCommand = ReactiveCommand.CreateFromTask(SetSubpixelIndicatorDecimals);
+
+        SetFastForwardSpeedCommand = ReactiveCommand.CreateFromTask(SetFastForwardSpeed);
+        SetSlowForwardSpeedCommand = ReactiveCommand.CreateFromTask(SetSlowForwardSpeed);
 
         // Context
         ToggleCommentsCommand = ReactiveCommand.Create(ToggleComments);
 
+        ShowInputDialog = new Interaction<InputDialogWindowViewModel, object>();
```

Comment:
What's this interaction for?  Generally an `Interaction` is so that the viewmodel can tell the view to do something.

Reply:
Seems like I forgot to remove that after moving everything to the DialogService.